content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -11,6 +11,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`interactiveTransactions`](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) since release [2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0) - [Submit feedback](https://github.com/prisma/prisma/issues/8664)
 - [`fullTextSearch`](/concepts/components/prisma-client/full-text-search) since release [2.30.0](https://github.com/prisma/prisma/releases/tag/2.30.0) - [Submit feedback](https://github.com/prisma/prisma/issues/8877)
 - [`referentialIntegrity`](/concepts/components/prisma-schema/relations/referential-integrity) since release [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1) - [Submit feedback](https://github.com/prisma/prisma/issues/9380)
+- [`dataProxy`](/concepts/data-platform/data-proxy) since release [3.3.0](https://github.com/prisma/prisma/releases/tag/3.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/9841)
```

Comment:
I think this somehow popped up here again but should not.

(Makes me worry there are other things)